devtools/x/build.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() {
+    println!("cargo:rustc-env=APTOS_TELEMETRY_DISABLE={}", 1);
```

Comment:
nit: can we add a comment to this? I'm assuming it disables telemetry for all `x` commands (i.e., just appends the env variable)? To an uninformed reader (like myself) this looks like magic 😄

Reply:
yup that's the intended effect. I'll document it as it's compiler magic to me as well :p